### PR TITLE
Adjust CoC reporting guidelines

### PR DESCRIPTION
**Pull request by jaimergp**
PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

As discussed in the last core meeting. cc @ocefpaf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ Thank you for helping make this a welcoming, friendly community for all.
 
 If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The team at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
 
+Reports should be sent to <conda-forge-core@googlegroups.com>, a private mailing list only accessible by the members of the core team. In your report involves a member of the core team, please send it to NumFOCUS following [these instructions](https://numfocus.org/code-of-conduct#how-to-report).
```
Comment (mfansler) [inline README.md:46]:
```suggestion
Reports should be sent to <conda-forge-core@googlegroups.com>, a private mailing list only accessible by the members of the core team. If your report involves a member of the core team, please send it to NumFOCUS following [these instructions](https://numfocus.org/code-of-conduct#how-to-report).
```

**Review by ocefpaf — APPROVED** [review]
LGTM. @jakirkham can you take a quick look and check if this reflects the outcome of our meeting with Katrina?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,8 @@ Thank you for helping make this a welcoming, friendly community for all.
 
 If you believe someone is violating the code of conduct, please report this in a timely manner. Code of conduct violations reduce the value of the community for everyone. The team at conda-forge takes reports of misconduct very seriously and is committed to preserving and maintaining the welcoming nature of our community.
 
+Reports should be sent to <conda-forge-core@googlegroups.com>, a private mailing list only accessible by the members of the core team. If your report involves a member of the core team, please send it to NumFOCUS following [these instructions](https://numfocus.org/code-of-conduct#how-to-report).
```
Comment (jakirkham) [inline README.md:46]: Yep this is what we discussed 👍

**Review by jakirkham — COMMENTED** [review]
LGTM. Thanks Filipe! 🙏

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Sorry somehow missed the approve button 🤦‍♂️

**Pull request merged**